www_src/components/range/range.jsx

```diff
@@ -16,13 +16,16 @@ var Range = React.createClass({
       value: 100
     };
   },
+  onChange: function (e) {
+    this.valueLink.requestChange(parseFloat(e.target.value));
```

Comment:
This change allows us to always return an actual number from ranges, instead of a string -- necessary for prop type validation